OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -760,6 +771,23 @@ static bool Team(S server, Connection conn, Session.Client client, string s)
 			return true;
 		}
 
+		static bool ToggleSpawn(S server, Connection conn, Session.Client client, string s)
+		{
+			if (!client.IsAdmin) return true;
```

Comment:
Non-admins can toggle spawns?

Reply:
This particular code path isn't called because the command never gets issued, having said that I was not fully aware of the intent of the bool. I have flipped it to return false. The answer is no they can't.